blocks/scripts/plot.py

```diff
@@ -0,0 +1,188 @@
+
+from __future__ import division, print_function
+
+import dill
+import fnmatch
+import logging
+import readline
+import pandas
+
+from six import iteritems
+from six.moves import input
+from collections import OrderedDict
+from functools import reduce
+
+from blocks import config
+from blocks.utils import change_recursion_limit
+
+
+def match_channel_specs(experiments, channel_specs):
```

Comment:
This is a very nice function which might be useful outside this script. Can you write a few tests for it?